src/index.js

```diff
@@ -186,31 +147,41 @@ const ATD_DocklessMap = (function() {
       });
     });
 
-    function postCellTripCount(feature, divId = "dataPane") {
-      var trip_percent =
+    function renderCellTripCount(feature, divId = "js-data-pane") {
```

Comment:
changed the name of this function just because post makes me think of POST requests to servers, not adding elements to the page.